class TaskException(Exception):
    pass


class HandleTaskException(TaskException):
    pass
